src/coreclr/pal/src/misc/tracepointprovider.cpp

```diff
@@ -62,10 +66,12 @@ PAL_InitializeTracing(void)
     // Check if loading the LTTng providers should be disabled.
     // Note: this env var is formally declared in clrconfigvalues.h, but
     // this code is executed too early to use the mechanics that come with that definition.
-    char *disableValue = getenv("COMPlus_LTTng");
-    if (disableValue != NULL)
+    CLRConfigNoCache cfgLTTng = CLRConfigNoCache::Get("LTTng", /*noprefix*/ false, &getenv);
+    if (cfgLTTng.IsSet())
     {
-        fShouldLoad = strtol(disableValue, NULL, 10);
+        DWORD value;
+        if (cfgLTTng.TryAsInteger(10, value))
```

Comment:
This one can be base 16 too. The only valid input to change behavior is 0.

Reply:
Agreed, but I am matching behavior at present so leaving it as is.